shared-test/src/main/java/uk/gov/di/authentication/sharedtest/extensions/SqsQueueExtension.java

```diff
@@ -90,7 +90,9 @@ public void beforeAll(ExtensionContext context) {
                         "{0}-{1}",
                         context.getTestClass().map(Class::getSimpleName).orElse("unknown"),
                         queueNameSuffix);
-        queueUrl = getQueueUrlFor(queueName).orElseGet(() -> createQueue(queueName));
+        var truncatedQueueName = queueName.substring(0, Math.min(80, queueName.length()));
+        queueUrl =
+                getQueueUrlFor(truncatedQueueName).orElseGet(() -> createQueue(truncatedQueueName));
         sqsClient.purgeQueue(PurgeQueueRequest.builder().queueUrl(queueUrl).build());
```

Comment:
This change is because the generated queue name for one of the tests exceeded the max permitted, which localstack did not previously enforce